### DISCUSSION TITLE
User poll: Devices?

### DISCUSSION THREAD

**Discussion by Exholips** [e-m-b-a/emba#1044] [Polls]
Which devices are you using emba with? What scenarios? What has yielded best results so far?

[a comment by an automated account is not shown]

**Comment by m-1-k-3** [comment]
Looks as this would make more sense in the discussions section here: https://github.com/e-m-b-a/emba/discussions

**Comment by m-1-k-3** [comment]
Moved this now to a discussion.
What do you mean exactly? Which target firmware or which box that you are running EMBA on?
